packages/xo-server-backup-reports/package.json

```diff
@@ -36,6 +36,7 @@
     "node": ">=6"
   },
   "dependencies": {
+    "@xen-orchestra/defined": "^0.0.0",
```

Comment:
why its version is `0.0.0`. Shouldn't we upgrade when we're releasing a new package ?

Reply:
It's the current version of the package.